components/counter-bar/src/index.js

```diff
@@ -6,24 +6,23 @@ import ResultCountTitle from '@govuk-frederic/result-count-title';
 
 import { BLACK, GREY_1, GREY_3, LINK_COLOUR, RED, WHITE, YELLOW } from 'govuk-colours';
 
-const Container = styled('div')({
+const Wrapper = styled('div')({
   display: 'flex',
+  flexWrap: 'nowrap',
   lineHeight: '1',
 });
 
-const CountWrapper = styled('div')(
+const TotalWrapper = styled('a')(
```

Comment:
this element shouldn't really be a plain `a`

in actual use, we'll likely want this to be a react-router-dom `Link` component - however that's not what it should be here

to achieve this, we'd want to pass in the component to use - and default that to be a plain `a`

there may be examples of how to do that kind of thing in `govuk-react`

Reply:
I've had a crack at implementing this! Please just let me know any thoughts on outstanding issues.